### ISSUE TITLE
AttributeError: module 'torch.jit' has no attribute '_script_if_tracing'

### ISSUE THREAD

**Issue by BlaineHerring** [pytorch/pytorch#59528]
## 🐛 Bug

I am getting this error any time I try to call any torchvision modules

## To Reproduce

1.import torchvision
2. print(torchvision.__version__)

## Expected behavior

I should get a version number

## Environment

Please copy and paste the output from our
[environment collection script](https://raw.githubusercontent.com/pytorch/pytorch/master/torch/utils/collect_env.py)
(or fill out the checklist below manually).

You can get the script and run it with:
import torchvision
print(torchvision.__version__)
```

 - PyTorch Version (e.g., 1.0): 1.4.0
 - OS (e.g., Linux): Windows 10
 - How you installed PyTorch (`conda`, `pip`, source): pip install torchvision
 - Build command you used (if compiling from source):
 - Python version: 3.8.10

**Comment by tugsbayasgalan** [comment]
Did you install the latest torchvision?

**Comment by BlaineHerring** [comment]
Yes I tried installing the latest version and several earlier versions. Nothing has worked.

**Comment by janeyx99** [comment]
TorchScript has been deprecated in favor of torch.export as of 2.11. https://pytorch.org/blog/pytorch-2-11-release-blog/ Closing this issue as we are not further developing on it--please ping if you believe this issue was erroneously closed.
